src/test/java/lotto/service/LottoServiceTest.java

```diff
@@ -0,0 +1,29 @@
+package lotto.service;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import camp.nextstep.edu.missionutils.test.NsTest;
+import java.util.List;
+import lotto.domain.Lotto;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class LottoServiceTest extends NsTest {
+
+    private final LottoService lottoService = new LottoService();
+
+    @DisplayName("로또가 요청된 개수만큼 생성되었는지 확인한다.")
+    @Test
+    void 로또_개수_일치_확인() {
+        int lottoNum = 3;
+        List<Lotto> lottos = lottoService.createLottos(lottoNum);
+
+        assertThat(lottoNum).isEqualTo(lottos.size());
```

Comment:
`assertThat(실제값).isEqualTo(기대값)` 형식으로 검증을 진행하는데, 해당 문장은 서로 뒤바뀐 것 같습니다.
`lottoService.createLottos(lottoNum)`의 결과로 나온 `lottos`의 `size()`를 확인하는 것이므로,
아래와 같이 수정하면 테스트의 의도를 더 잘 드러낼 수 있을 것 같아요!
```suggestion
        assertThat(lottos.size()).isEqualTo(lottoNum);
```